testing/tests/DevExpress.ui.widgets.editors/checkbox.tests.js

```diff
@@ -482,6 +482,47 @@ QUnit.module('Checkbox', function() {
                     assert.notOk(instance._isFocused(), 'checkBox is not focused');
                 });
             });
+
+            QUnit.module('indeterminate state', function() {
+                QUnit.test('click triggers user handler and changes state from false to null, from null to true', function(assert) {
+                    const $element = $('#checkBox').dxCheckBox({
+                        indeterminateStateEnabled: true,
+                        focusStateEnabled: true,
+                        value: false
```

Comment:
Let's check the unsupported value as 'string' and various combination of initial value, not only the `false` value

Reply:
https://github.com/DevExpress/DevExtreme/pull/22616/commits/203d9e5c4641945c4a7c51fbb008c84a9a881652